src/xcode/ENA/ENA/Source/Scenes/Settings/ResetViewController.swift

```diff
@@ -36,6 +36,7 @@ final class ResetViewController: UIViewController {
 
 	@IBAction func resetData(_: Any) {
 		delegate?.reset()
+		dismiss(animated: true, completion: nil)
```

Comment:
After the reset this dismissing shouldn't be needed.

Reply:
If i remove it the Reset screen is not automatically dismissed and stays on top of the Onboarding Workflow until it is dismissed manually. 
This happens on all of my test devices so I would leave it in.

Reply:
Yes, that is what I noticed too.